presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java

```diff
@@ -78,7 +78,11 @@ else if (usingTeradataJdbcDriver(connection)) {
 
     public static boolean usingPrestoJdbcDriver(Connection connection)
     {
-        return getClassNameForJdbcDriver(connection).equals("com.facebook.presto.jdbc.PrestoConnection");
+        String canonicalClassName = getClassNameForJdbcDriver(connection);
```

Comment:
I'd leave it as is given is falling into the same pattern as pre-change behavior.